openmc/material.py

```diff
@@ -24,6 +25,25 @@
 DENSITY_UNITS = ('g/cm3', 'g/cc', 'kg/m3', 'atom/b-cm', 'atom/cm3', 'sum',
                  'macro')
 
+_MAT_LIB_DIRS = [openmc.data.INTERNAL_DATA_PATH / 'material_libraries']
+if os.environ.get('OPENMC_MATERIAL_LIBRARY_PATH') is not None:
+    _MAT_LIB_DIRS += [Path(a) for a in os.environ.get('OPENMC_MATERIAL_LIBRARY_PATH').split(':')]
+
+_MAT_LIB_PATHS = [f.resolve() for p in _MAT_LIB_DIRS
+                  for f in sorted(Path(p).glob('*'))]
+
+MATERIAL_LIBRARIES = {}
```

Comment:
The difference between the types of values in `MATERIAL_LIBRARIES` and `_MATERIAL_LIBRARY_CACHE` wasn't clear to me until reading the code. Can we add some comments on the types of keys and values in each?

Reply:
done